packages/file_selector/file_selector_platform_interface/lib/src/types/x_type_group/x_type_group.dart

```diff
@@ -1,27 +1,26 @@
 // Copyright 2013 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
+import 'package:meta/meta.dart';
```

Comment:
This should be

```
import 'package:flutter/foundation.dart' show immutable;
```

to avoid needing an extra dependency.

Reply:
The package does not depend (directly) on `package:meta`, and it wasn't added to the pubspec in this PR. I think we need this lint:

https://dart-lang.github.io/linter/lints/depend_on_referenced_packages.html

Reply:
We could potentially add the lint, but `publishable` catches this mistake.